Library/Strings.swift

```diff
@@ -19,7 +19,7 @@ public enum Strings {
    - **fr**: "Environ %{reward_amount}"
    - **en**: "About %{reward_amount}"
   */
-  public static func About_reward_amount(reward_amount reward_amount: String) -> String {
+  public static func About_reward_amount(reward_amount: String) -> String {
```

Comment:
lol at the diff in this file. lucky it's all computer generated, we just have to fix the [`bin/strings.swift`](https://github.com/kickstarter/ios-oss/blob/master/bin/strings.swift) to be swift 3 compatible.